src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -97,8 +98,37 @@ export default function ConfirmedAppointmentListItem({
         )}
       </div>
 
+      {!allowCancel &&
+        !canceled && (
+          <div className="vads-u-margin-top--2">
+            <button
```

Comment:
It might make sense to wrap this button in a separate `AddToCalendar` component. We'll also likely want to use it on the confirmation page.

Reply:
Done